.github/workflows/e2e-tests.yml

```
@@ -44,15 +48,21 @@ jobs:
       env:
         NGROK_KEY: ${{ secrets.NGROK_KEY }}
         CONFIG_KEY: ${{ secrets.E2E_CONFIG_KEY }}
+        LATEST_GUTENBERG: ${{ matrix.gutenberg == 'latest' }}
```

This winds up setting LATEST_GUTENBERG to "true" or "false", I believe, while the script looks for empty versus non-empty. See at https://github.com/Automattic/jetpack/pull/17950/checks?check_run_id=1487874826#step:7:28 it's installing Gutenberg anyway.

The most straightforward fix would probably be to change this to just
```suggestion
        GUTENBERG: ${{ matrix.gutenberg }}
```
and have the script do the comparison instead, like
```bash
if [ "$GUTENBERG" == "latest" ]; then
```

There's also another reference to LATEST_GUTENBERG in ./tests/e2e/lib/reporters/slack.js that might also need to be updated.

BTW, does that slack.js reporter get called anywhere below where the environment variable would need to be set? Or if you're unsure you could just move the `env` for it to the job level rather than having it on this one step.

---

> This winds up setting LATEST_GUTENBERG to "true" or "false", I believe, while the script looks for empty versus non-empty. See at https://github.com/Automattic/jetpack/pull/17950/checks?check_run_id=1487874826#step:7:28 it's installing Gutenberg anyway.

Ah, great spot, thanks a lot! Sloppy me only checked the with-GB job to see if it's being installed :sweat_smile: 

> The most straightforward fix would probably be to change this to just [...] and have the script do the comparison instead, like
> 
> ```shell
> if [ "$GUTENBERG" == "latest" ]; then
> ```

Yep, sounds good, I'll do that :+1: 

> There's also another reference to LATEST_GUTENBERG in ./tests/e2e/lib/reporters/slack.js that might also need to be updated.

:+1: 

> BTW, does that slack.js reporter get called anywhere below where the environment variable would need to be set? Or if you're unsure you could just move the `env` for it to the job level rather than having it on this one step.

I'll have a look :+1:

---

> BTW, does that slack.js reporter get called anywhere below where the environment variable would need to be set? Or if you're unsure you could just move the `env` for it to the job level rather than having it on this one step.

Seems like the slack reporter [is used during global teardown](https://github.com/Automattic/jetpack/blob/07a725146df465cc8df0f08adaa840cd41a4f582/tests/e2e/lib/global-teardown.js#L4) (e2e). I think this would affect at least the with-Gutenberg and without-Gutenberg e2e test steps. Seems like a move to job level would be a good idea :+1: 

(Generally, I'd love if we didn't need to rely on env vars so implicitly, but that's outside of the scope of this PR.)